### PR TITLE
hlx publish warns when deleting keys from dictionary

### LINKED ISSUE #416: hlx publish warns when deleting keys from dictionary
https://github.com/adobe/helix-cli/issues/408#issuecomment-449047430

### PR DESCRIPTION
**Pull request by rofe**
Fixes #416

### CODE REVIEW

**Comment by rofe** [comment]
Ok, thanks for the info. In the meantime, this still helped me reduce the noise in `hlx publish` output :)

Feel free to close this PR and delete the branch.

**Comment by rofe** [comment]
Btw, any idea why we have "850 new vulnerable dependencies" all over sudden? Is this a glitch in snyk? When I build `helix-cli` locally, I get 0.

**Comment by rofe** [comment]
@tripodsan: I created https://github.com/adobe/helix-publish/issues/8

Like @kptdobe, I would still consider this change useful though.

File: src/publish.cmd.js [hunk 1]
```diff
@@ -442,7 +442,7 @@ class PublishCommand extends AbstractCommand {
       const opts = await this.deleteOpts(`/dictionary/${mydict}/item/${key}`);
       await request(opts);
     } catch (e) {
-      this.log.error(`Unknown error when deleting key ${key} from dictionary ${mydict}`, e);
+      this.log.debug(`Unknown error when deleting key ${key} from dictionary ${mydict}`, e);
```
[the remark this answers is not in the document]
Comment (trieloff) [reply src/publish.cmd.js]: helix-publish is solving this differently, so this fix is ok for now and does not need to get ported.

**Pull request merged**